packages/mdc-switch/README.md

```diff
@@ -59,16 +59,27 @@ npm install --save @material/switch
 <label for="another-basic-switch" class="mdc-switch-label">off/on</label>
 ```
 
-## Classes
+### Classes
 
-### Block
+#### Block
 
 The block class is `mdc-switch`. This defines the top-level switch element.
 
-### Modifier
+#### Modifier
 
 The provided modifiers are:
 
-| Class                 | Description                                  |
-| ----------------------| -------------------------------------------- |
-| `mdc-switch--disabled`   | Applies disabled style to disabled switches. |
+| Class                  | Description                                  |
+| -----------------------| -------------------------------------------- |
+| `mdc-switch--disabled` | Applies disabled style to disabled switches. |
+
+### Sass Mixins
+
+The following mixins apply only to _enabled_ switches in the _on_ (checked) state.
+It is not currently possible to customize the color of a _disabled_ or _off_ (unchecked) switch.
+
+Mixin | Description
+--- | ---
+`mdc-switch-checked-track-color($color)` | Sets the track color
```

Comment:
Should we just make this `mdc-switch-track-color($color)`? Simplify our naming. (and the same for the other two)

Reply:
Done